usagov/vote-gov-drupal
# VOTE-744 & 745 mega menu configuration

## Jira ticket

https://cm-jira.usa.gov/browse/VOTE-744
https://cm-jira.usa.gov/browse/VOTE-745

## Description

Configuration to allow editors to manage the custom content in the mega menus

## Deployment and testing

### Post-deploy

1. `lando retune`

### QA/Test

1. log in and visit http://vote-gov.lndo.site/admin/structure/menu/manage/main
2. Delete the Homepage link for all languages
3. Edit the Register to Vote link and add some placeholder text in the new content field
4. visit http://vote-gov.lndo.site/admin/structure/menu/manage/main/view_modes_settings/default and select megamenu view mode for the register to vote link
5. visit http://vote-gov.lndo.site/ and confirm the dropdown icon displays and that you can click the "Register to vote" text to expand the megamenu with the placeholder content inside (ignore styling of the megamenu for this task)

## Checklist for the Developer




## Checklist for the Peer Reviewers
